runtime/mount/endpoints/run.py

```diff
@@ -164,10 +179,21 @@ async def run(s: Span, ctx: Context) -> HandlerResult:
                 k_proc.proc.send_signal(signal=signal.SIGINT)
                 continue
 
+            if msg["type"] == "run_cell" and not is_session_owner:
+                await ctx.send_message(
+                    orjson.dumps(
+                        {
+                            "type": "error",
+                            "data": {"message": "user is not session owner"},
+                        }
+                    ).decode()
+                )
+                continue
+
             await conn_k.send(msg)
     except WebsocketConnectionClosedError:
         ...
     finally:
```

Comment:
should move the `ready_ev.wait` and the ready message into the `try` block in case they throw